### DISCUSSION TITLE
Organisation Private Repositories

### DISCUSSION THREAD

**Discussion by vultuk** [community/community#4892] [Codespaces] [answered]
Just wondering what kind of time scale we are looking at before we will be able to test out codespaces on private repositories inside organisations. 99.9% of the work I do is inside private repositories on my organisations and I'm unable to beta test because of this.

**Comment by 2percentsilk** [comment, marked as answer]
Hi @vultuk,

If you're primarily working within an organization, you may be a good candidate for our organization specific beta. I've reached out via email to chat more!
